2.segmentation/README.md

```diff
@@ -0,0 +1,9 @@
+# Image segementation of objects
+
+We need to segment the images into objects.
```

Comment:
Recommend including what objects will be segmented (e.g., segmenting individual bones per fish).

Reply:
addressed